edb/pgsql/ast.py

```diff
@@ -884,22 +878,21 @@ def copyfrom(self, other):
 
 
 class SubLinkType(enum.IntEnum):
-    EXISTS = enum.auto()
-    NOT_EXISTS = enum.auto()
-    ALL = enum.auto()
-    ANY = enum.auto()
-    EXPR = enum.auto()
+    pass
+    pass
+    pass
+    pass
```

Comment:
Maybe just one `pass`.

Reply:
Wait, no, maybe it can just go away :)

Reply:
Maybe I should start re-reading my commits before pushing :D